typings/src/globals/globals.ts

```diff
@@ -0,0 +1,98 @@
+import { hrTestData, IHRTestData } from './test_data/test_data';
+
+interface IApp {
+    financials: {
+        adminUserId: string;
+        loginPassword: string;
+    };
+}
+
+const appConfig: IApp = {
+    financials: {
+        /** Financials admin user id for logging into financials */
+        adminUserId: `${process.env.FINANCIALS_ADMIN_USER_ID}`,
+        /** Password used for logging into financials. */
+        loginPassword: `${process.env.FINANCIALS_TEST_PASSWORD}`
+    }
+};
+
+interface IBrowserUrlConfig {
+    financialsLogin: string;
+    financialsRootUrl: string;
+}
+const browserUrlConfig: IBrowserUrlConfig = {
+    financialsLogin: `${process.env.AUTH_PROTOCOL}://${process.env.FINANCIALS_ENTITY}.fms.${
+        process.env.FINANCIALS_TEST_ENV
+    }/login.aspx`,
+    financialsRootUrl: `${process.env.AUTH_PROTOCOL}://${process.env.FINANCIALS_ENTITY}.fms.${
+        process.env.FINANCIALS_TEST_ENV
+    }/stw_php/`
```

Comment:
rename `financialsRootUrl` to `coinflexRootUrl` and change the value too `${process.env.AUTH_PROTOCOL}://${process.env.INSTANCE_NAME}.coinflex.com`